### PR TITLE
Replaced WebL10n in Abecedarium

### PR DESCRIPTION
**Pull request by Vishalk91-4**
Issue #1378 
Replaced WebL10n in Abecedarium

The tutorial is working here
image

@llaske could you please review it

### CODE REVIEW

File: activities/Abecedarium.activity/lib/l10n.js [hunk 1]
```diff
@@ -0,0 +1,58 @@
+define(['i18next.min', 'axios.min'], function (i18next, axios) {
+    const l10n = {};
+    let initialized = false;
+
+    l10n.init = async (lang) => {
+        await i18next.init({
+            lng: lang,
+            debug: true,
```
Comment (llaske) [inline activities/Abecedarium.activity/lib/l10n.js:8]: Will be better to disable i18next debug information.  Was not there in the initial file https://github.com/llaske/sugarizer/blob/dev/activities/QRCode.activity/lib/l10n.js#L6

**Comment by Vishalk91-4** [comment]
@llaske, I've disabled it

**Comment by llaske** [comment]
There is still some references to webL10n library in the code.
It could cause some issues, for example here at the end of the play game:

image

Could you ensure the library is replaced everywhere?

**Comment by Vishalk91-4** [comment]
@llaske 
Now I've checked them all, and there is no occurrence of webL10n in Abecedarium
image

**Comment by llaske** [comment]
Nice. Thanks.

**Pull request merged**